code/modules/reagents/chemistry/reagents/disease.dm

```diff
@@ -121,6 +130,7 @@
 	reagent_state = LIQUID
 	color = "#F7E6B1"
 	can_synth = FALSE
+	taste_description = "bacon"
```

Comment:
normally we don't want to be able to taste poisons, but bacon is a flavor that exists elsewhere, so to mistake this poison for something that tastes good actually makes it even stealthier.